ansible/playbooks/tasks/azure-cluster-hana.yaml

```diff
@@ -29,6 +31,22 @@
   ansible.builtin.command: crm configure primitive rsc_SAPHanaTopology_HDB_HDB00 ocf:suse:SAPHanaTopology operations \$id="rsc_sap2_HDB_HDB00-operations" op monitor interval="10" timeout="600" op start interval="0" timeout="600" op stop interval="0" timeout="300" params SID="HDB" InstanceNumber="00" 
   when: hana_topology_resource | length == 0
 
+- name: Set stonith-timeout [sdb]
+  ansible.builtin.command: crm configure property stonith-timeout=144
+  when:
+    - stonith_timeout | length == 0
+    - use_sbd | bool
+  
+- name: Set stonith-timeout [azure fencing]
+  ansible.builtin.command: crm configure property stonith-timeout=900
+  when:
+    - stonith_timeout | length == 0
+    - use_sbd | bool
+
+- name: Enable stonith
+  ansible.builtin.command: crm configure property stonith-enabled=true
+  when: stonith_enabled != 'true'
+
 - name: Create HANA topology clone
```

Comment:
Ensures that the rebuild of the SBD STONITH resources only occurs when using SBD